src/svea_sensors/launch/transforms.launch

```diff
@@ -7,7 +7,8 @@
     <arg name="is_indoors"      default="true" />
 
     <node pkg="tf2_ros" type="static_transform_publisher" name="odommap_broadcaster"        args="0 0 0 0 0 0 map odom" />
-    <node pkg="tf2_ros" type="static_transform_publisher" name="imulink_broadcaster"        args="0.0 0.0 0.1 0 0 0 base_link imu" />
+    <node pkg="tf2_ros" type="static_transform_publisher" name="baselinkodom_broadcaster"   args="0 0 0 0 0 0 odom base_link" />
```

Comment:
Why does odom->baselink does have a **static** transform publisher?

Reply:
For the localization debugger, we start EKF nodes without publishing `tf`, and then `navsat_transform_node` won't initialize as it cannot find the transform between `map` and `base_link`. Hence this static transform is added. However, this should be harmless when EKF nodes do publish `tf` as `/tf` has priority over `/tf_static`, and hence this static transform will expire as soon as a new `map->baselink` transform becomes available.